src/uint/shl.rs

```diff
@@ -5,6 +5,7 @@ use core::ops::{Shl, ShlAssign};
 
 impl<const LIMBS: usize> Uint<LIMBS> {
     /// Computes `self << 1` in constant-time, returning the overflowing bit as a `CtChoice`.
+    #[allow(unused)]
```

Comment:
Since it's not public, I think we can safely remove it if it's unused

Reply:
Updated now